pages/base.page.js

```diff
@@ -0,0 +1,38 @@
+const PAGE_TIMEOUT = 10000
+
+export default class BasePage {
+
+
+   /**
+    * Abrir página
+    * @param {String} ruta a la cual acceder
+    */
+   async abrir(ruta) {
+       await browser.url(`${ruta}`);
```

Comment:
As you are not really building a string based on interpolation here, you can simplify this to just:

```javascript
await browser.url(ruta);
```